src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj

```diff
@@ -72,8 +72,9 @@
   </ItemGroup>
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\..\..\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj" />
-    <ProjectReference Include="..\..\..\EditorFeatures\Core\Microsoft.CodeAnalysis.EditorFeatures.csproj" />
-    <ProjectReference Include="..\..\..\EditorFeatures\Core.Wpf\Microsoft.CodeAnalysis.EditorFeatures.Wpf.csproj" />
+    <!-- We do not publish EditorFeatures.Common or EditorFeatures.WPF to NuGet -->
```

Comment:
```suggestion
    <!-- We do not publish Microsoft.CodeAnalysis.EditorFeatures or Microsoft.CodeAnalysis.EditorFeatures.Wpf to NuGet, so mark them PrivateAssets="all" -->
```

Do we still have some other file that controls this fact? Worrying what happens if we started uploading them, would we remember to update this?